exporter/sumologicexporter/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sumolo
 go 1.14
 
 require (
+	github.com/klauspost/pgzip v1.0.2-0.20170402124221-0bf5dcad4ada
```

Comment:
This is rather old. Can we:

* use a newer version?
* use a concrete tag: I suggest https://github.com/klauspost/pgzip/releases/tag/v1.2.5

?